pkg/manifest/manifest_writer.go

```diff
@@ -128,6 +130,17 @@ func toWriteableEnvironmentGroups(environments map[string]EnvironmentDefinition)
 	return result
 }
 
+func getType(env EnvironmentDefinition) string {
+	switch env.Type {
+	case Classic:
+		return "classic"
+	case Platform:
+		return "platform"
+	}
+
+	panic(fmt.Sprintf("Unexpected environment type %q in environment %q.", env.Type, env.Name))
```

Comment:
i think that's not the right place to panic

Reply:
I missed that! Sry. Agree with @warber if we have an error case here we should return and handle errors as everywhere else - even if it's a "this should never happen" case like I think this one is